uktrade/data-hub-frontend
# Refactor/opportunity detail form

## Description of change

Refactors the _investment opportunity details_ and _requirement_ forms with the `TaskForm` component.

## Test instructions

Everything should work as before, but here are the instructions:

1. Go to `/investments/opportunities/{id}/details`
2. Expand the _Opportunity details_ section of the accordion
3. Click the _Edit_ button
4. Change the values in the form
5. Click the _Save_ button
6. The form should be replaced by a view showing the updated values
7. Expand the _Opportunity requirements_ tab
8. Repeat steps 3 to 6

## Screenshots
### Before
<img width="616" alt="Screen Shot 2021-11-18 at 1 29 24 PM" src="https://user-images.githubusercontent.com/2333157/142426021-711a4f20-7801-4443-927c-dd39c34bc51e.png">

### After
<img width="662" alt="Screen Shot 2021-11-18 at 1 31 19 PM" src="https://user-images.githubusercontent.com/2333157/142426012-bafb7cfa-fcc5-43ad-818e-095fc697912b.png">
